setup.py

```diff
@@ -50,6 +50,7 @@
                                            "rulesets/fedora.json"]),
                 ("share/bash-completion/completions/", ["bash-complete/colin"])],
     license='GPLv3+',
+    python_requires=">=3.5",
```

Comment:
```suggestion
    python_requires=">=3.6",
```

Reply:
I do not have preferences about 3.5 vs 3.6
 but +1 for for adding requirements >3.x

Reply:
Good spot, this should be 3.6. I'll update the classifiers too.